components/brave_ads/core/internal/BUILD.gn

```diff
@@ -666,6 +666,23 @@ static_library("internal") {
     "flags/environment/environment_flag_util.h",
     "flags/flag_constants.h",
     "flags/flags_util.cc",
+    "geographical/region/region.cc",
+    "geographical/region/region.h",
+    "geographical/subdivision/subdivision.cc",
+    "geographical/subdivision/subdivision.h",
+    "geographical/subdivision/subdivision_feature.cc",
+    "geographical/subdivision/subdivision_feature.h",
+    "geographical/subdivision/subdivision_observer.h",
+    "geographical/subdivision/subdivision_url_request.cc",
+    "geographical/subdivision/subdivision_url_request.h",
+    "geographical/subdivision/subdivision_url_request_builder.cc",
+    "geographical/subdivision/subdivision_url_request_builder.h",
+    "geographical/subdivision/subdivision_url_request_builder_util.cc",
+    "geographical/subdivision/subdivision_url_request_builder_util.h",
+    "geographical/subdivision/subdivision_url_request_constants.h",
+    "geographical/subdivision/subdivision_url_request_delegate.h",
+    "geographical/subdivision/subdivision_url_request_json_reader_util.cc",
+    "geographical/subdivision/subdivision_url_request_json_reader_util.h",
```

Comment:
What do you think about moving these files to `common/geographical`?

Reply:
moved common/region to common/country_code